src/content/reference/react-dom/createPortal.md

```diff
@@ -20,11 +20,11 @@ title: createPortal
 
 ---
 
-## Reference {/*reference*/}
+## Başvuru dokümanı {/*reference*/}
 
 ### `createPortal(children, domNode, key?)` {/*createportal*/}
 
-To create a portal, call `createPortal`, passing some JSX, and the DOM node where it should be rendered:
+Bir portal oluşturmak için, `createPortal` çağrısını yapınız, bazı JSX'leri ve işlenmesi gereken DOM düğümünü iletiniz:
```

Comment:
Buradaki `some` `bazi` dan ziyade `biraz, bir miktar` anlaminda. Cumleyi de biraz toparlayacak olursak asagidaki daha iyi bir ceviri olabilir:

Bir portal oluşturmak için, biraz JSX ve render edilmesi gereken DOM düğümünü ileterek `createPortal`ı çağırın: